library/mobjects.c

```diff
@@ -274,6 +274,19 @@ static int eblob_base_ctl_open(struct eblob_backend *b, struct eblob_base_ctl *c
 	} else {
 		struct stat st;
 
+		if (created) {
+			eblob_log(b->cfg.log, EBLOB_LOG_INFO,
+				  "bctl: index: %d: removing sorted index that shouldn't exist: %s\n", ctl->index, full);
+
+			if (unlink(full) == -1) {
+				err = -errno;
+				EBLOB_WARNC(b->cfg.log, EBLOB_LOG_ERROR, errno, "unlink: %s", full);
```

Comment:
This should tell more than just unlink error, like `failed to remove sorted index X that should not exist because we have just created a new blob with that name: error string, error code`

The same at above info log line, please explain, why it should not exist